packages/tasks/src/model-libraries.ts

```diff
@@ -590,6 +590,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			OR path:"open_clip_pytorch_model.bin"
 			OR path:"pytorch_model.bin"`,
 	},
+	"open-sora": {
+		prettyLabel: "OpenSora",
+		repoName: "OpenSora",
```

Comment:
```suggestion
		prettyLabel: "Open-Sora",
		repoName: "Open-Sora",
```

(based on https://github.com/hpcaitech/Open-Sora's README)